coderd/database/querier_test.go

```diff
@@ -2164,11 +2171,125 @@ func TestGetProvisionerJobsByIDsWithQueuePosition(t *testing.T) {
 		t.SkipNow()
 	}
 
+	now := dbtime.Now()
+	ctx := testutil.Context(t, testutil.WaitShort)
+
+	testCases := []struct {
+		name           string
+		jobTags        []database.StringMap
+		daemonTags     []database.StringMap
+		queueSizes     []int64
+		queuePositions []int64
+	}{
```

Comment:
I really like this testCase structure :heart:

Reply:
I extended it with:

```go
// GetProvisionerJobsByIDsWithQueuePosition takes jobIDs as a parameter.
// If skipJobIDs is empty, all jobs are passed to the function; otherwise, the specified jobs are skipped.
// NOTE: Skipping job IDs means they will be excluded from the result,
// but this should not affect the queue position or queue size of other jobs.
skipJobIDs map[int]struct{}
```
to cover additional test cases.